content/about/about-content.md

```diff
@@ -28,8 +28,12 @@ A lot of the work in EEW stems from EDGI’s work around “Environmental Data J
 NAMES ETC
 
 ## Collaborators
-![EEW's collaborators: Stony Brook University Geospatial Center, Teaching Learning Lab, Sunrise Boston](./collabs.png)
+<a href="https://www.stonybrook.edu/commcms/gss/" target=_blank >Geospatial Center at Stony Brook University</a>
+<a href="http://apps.tlt.stonybrook.edu/" target=_blank >Teaching Learning Lab at Stony Brook University</a>
```

Comment:
nice! I'm impressed you picked up my html formatting with `target=_blank` – this extra argument when added to an `a href` tells the browser to open the link in a new tab.